app/assets/stylesheets/views/_taxons.scss

```diff
@@ -35,6 +35,59 @@
   margin-top: 0;
 }
 
+.taxon-page__featured-item {
+  @extend %contain-floats;
+  margin-top: $gutter-two-thirds;
+
+  &--single {
+    min-width: 330px;
+  }
+
+  .taxon-page__featured-image {
+    @include box-sizing(border-box);
+    width: (1 / 3) * 100%;
+    float: left;
+    margin-bottom: $gutter-one-third;
+    padding-right: $gutter-half;
+
+    @include media(tablet) {
+      float: none;
+      margin-bottom: $gutter-one-third;
+      padding-right: 0;
+      width: 100%;
+    }
+  }
+
+  .taxon-page__featured-text {
+    display: inline-block;
+    width: (2 / 3) * 100%;
+
+    @include media(tablet) {
+      width: 100%;
+    }
+  }
+
+  .taxon-page__featured-link {
+    @include bold-19;
+  }
+
+  .taxon-page__featured-metadata-wrapper {
+    @include core-16;
+    margin-top: $gutter-one-third / 2;
+
+    .taxon-page__featured-metadata {
+      display: inline-block;
+      margin-right: $gutter-two-thirds;
+    }
+  }
+}
+
+.taxon-page__featured-see-more {
+  display: block;
```

Comment:
Suggest taking display block off this element and moving the margin to the bottom of the element above. Display block on links can be confusing for users, because if they click near the link they might hit it without intending to. Also because this seems to be the same (in appearance) link as the one that appears when there's a list of stories, and that isn't block.